src/Command/Anonymization/ConfigDumpCommand.php

```diff
@@ -48,7 +48,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 $io->table(
                     ['Target', 'Anonymizer', 'Options'],
                     \array_map(
-                        fn (AnonymizerConfig $config) => [
+                        fn (AnonymizerConfig $config): array => [
```

Comment:
We don't want this kind of type on arrow function, there is no really need to have it considering the actual use case

Reply:
The idea here is make strong type all returning values by default with PHP 8.3. Putting array ensures that always is an array and autocompleters cand deduce types in IDE (even if always is an array)

Reply:
Again, in this use case, this is not necessary. 

We don't follow the rector's rule on this project. Our coding style and rules are fixed by phpcs and we don't want to add more for now.

Thanks

Reply:
Done